sessionctx/variable/sysvar.go

```diff
@@ -386,7 +392,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal, "log_syslog_include_pid", ""},
 	{ScopeSession, "last_insert_id", ""},
 	{ScopeNone, "innodb_ft_cache_size", "8000000"},
-	{ScopeNone, "log_bin", "OFF"},
+	{ScopeGlobal | ScopeSession, LogBin, boolToStatusStr(config.GetGlobalConfig().Binlog.Enable)},
```

Comment:
I think this variable should not be a global one.

Reply:
we can set  `ScopeNode` to let this variable read only now.